mmedit/apis/restoration_face_inference.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import numpy as np
+import torch
+from mmcv.parallel import collate, scatter
+
+from mmedit.datasets.pipelines import Compose
+
+if torch.__version__ >= '1.7.0':
```

Comment:
mmcv has some utils that helps version comparison

Reply:
Done.